src/test/java/com/wikia/webdriver/elements/mercury/components/loginAndSignup/RegisterArea.java

```diff
@@ -0,0 +1,167 @@
+package com.wikia.webdriver.elements.mercury.components.loginAndSignup;
+
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.elements.mercury.components.TopBar;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class RegisterArea extends WikiBasePageObject {
+
+  @FindBy(css = ".auth.desktop.signin-page")
+  private WebElement authModal;
+  @FindBy(css = "#signupEmail")
+  private WebElement signupEmail;
+  @FindBy(css = "#signupUsername")
+  private WebElement signupUsername;
+  @FindBy(css = "#signupPassword")
+  private WebElement signupPassword;
+  @FindBy(css = "#signupBirthDate")
+  private WebElement signupBirthdate;
+  @FindBy(css = ".birth-month")
+  private WebElement signupBirthMonth;
+  @FindBy(css = ".birth-day")
+  private WebElement signupBirthDay;
+  @FindBy(css = ".birth-year")
+  private WebElement signupBirthYear;
+  @FindBy(css = "#signupSubmit")
+  private WebElement signupSubmitButton;
+  @FindBy(css = ".wikia-nav__avatar")
+  private WebElement avatar;
+  @FindBy(xpath = "//*[@id=\"signupForm\"]/div[1]/small")
+  private WebElement emailError;
+  @FindBy(css = "#signupForm div:nth-child(2) small")
+  private WebElement usernameError;
+  @FindBy(xpath = "//*[@id=\"signupForm\"]/div[3]/small")
+  private WebElement passwordError;
+  @FindBy(css = "#signupForm > small.error")
+  private WebElement genericError;
+  @FindBy(css = " header.auth-header")
+  private WebElement registerHeader;
+  @FindBy(css = ".loginAndSignup")
+  private WebElement newloginButton;
+  @FindBy(css = ".signup-provider-email")
+  private WebElement signupButton;
+
+//  private WebDriver driver;
+//  private Wait wait;
+  private UrlBuilder urlBuilder;
+
+  private final String mainWindowHandle;
+
+
+  public RegisterArea(WebDriver driver) {
+//    this.driver = driver;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)